db/patches/V_Unknown_ship_trader_changes.sql

```diff
@@ -0,0 +1,51 @@
+--Merchant Vessel -20  holds (170 -> 150).  Trade throughput down from 1530 (9tph * 170 cargo holds) to 1350 (9tph * 150 cargo holds).
```

Comment:
Before we merge any PR with database changes, we'll need to make sure the `.sql` files have a numeric version in the filename (for use with Flyway) instead of "Unknown". Not necessary to change right now though.